src/Widgets/Views/Editor.vala

```diff
@@ -546,7 +560,15 @@ namespace Webpin.Widgets.Views {
                 }
                 reset_grab_color_and_icon ();
                 update_app_icon ();
+                updateFormStatus();
             }
         }
     }
 }
+
+public void set_class(Gtk.Widget widget, string class_name, bool flag) {
+    if (flag)
+        widget.get_style_context().add_class(class_name);
+    else
+        widget.get_style_context().remove_class(class_name);
+}
```

Comment:
Why don't you include this method in the `Editor` class?

Reply:
I thought static was better - then we could move it to a utils file in case it's needed somewhere else :man_shrugging: